lib/ffmpeg/black_detect.rb

```diff
@@ -15,7 +15,7 @@ def valid?
 
     def run
       # ffmpeg will output to stderr
-      command = "#{@movie.ffprobe_command} -f lavfi -i \"movie=#{Shellwords.escape(@movie.path)},blackdetect[out0]\" -show_entries tags=lavfi.black_start,lavfi.black_end -of default=nw=1 -v quiet"
```

Comment:
Correct -- it's being done in the initializer rather than as each one is referenced (since it was messy and repetitive):
https://github.com/Vidyard/rlovelett-ffmpeg/blob/9d0e7236b5899678fe7dda99db07ca9d0a0f38bb/lib/ffmpeg/movie.rb#L20-L23